python/simulation/fake_raft.py

```diff
@@ -0,0 +1,575 @@
+"""
+Makes a raft's worth of image files based on a user-specified single-sensor
+image file, the raft designator, and the sensor type (ITL or E2V, which
+probably could be guessable from the single-sensor image).  The test type
+probably also needs to be specified (for the file naming)
+The data in the nine output images are the same as in the input image.  What
+is new is the raft coordinate-related header keywords.  These are derived
+from the specification in LCA-13501.
+"""
+
+from __future__ import print_function, absolute_import, division
+
+import os
+import fnmatch
+import yaml
+
+import astropy.io.fits as fits
+from datacat.error import DcClientException
+from DataCatalog import DataCatalog
+# import siteUtils
+
+
+def make_datacat_path(**kwargs):
+    """ Build the data catalog path for a particular test on a particular sensor
+
+    Looking at LSSTTD-690 it appear that with run numbers the best we can do is
+    <root_folder>/<sensor_type>/<sensor_id>.
+    """
+    return os.path.join(kwargs['root_folder'], kwargs['sensor_type'],
+                        kwargs['sensor_id'])
+
+
+def make_outfile_path(**kwargs):
+    """ Build the path for an output file for a particular test on a particular sensor
+
+    Looking at LSSTTD-690 for rafts this should be
+    <root_folder>/sraft<raft_id>/<process_name>/<job_id>/s<slot_name>/<file_string>
+    """
+    return os.path.join(kwargs['root_folder'], "sraft%s" % (kwargs['raft_id']),
+                        "%04i" % (kwargs['run_id']), kwargs['process_name'],
+                        "%04i" % (kwargs['job_id']), "s%s" % (kwargs['slot_name']),
```

Comment:
I think this should be 
```
                        "%04i" % (kwargs['job_id']), "S%s" % (kwargs['slot_name']),
```
i.e., an upper case 'S' instead of lower case, since that's what DM does.